internal/tiltfile/tiltfile_state.go

```diff
@@ -453,6 +455,7 @@ func (s *tiltfileState) OnStart(e *starkit.Environment) error {
 		{disableFeatureN, s.disableFeature},
 		{disableSnapshotsN, s.disableSnapshots},
 		{setTeamN, s.setTeam},
+		{telemetryCmdN, s.setTelemetryCmd},
```

Comment:
please don't add new built-ins here, make it an extension

Reply:
Got it, will change.

Reply:
LMK what you think now @nicks!